tokio-console/README.md

```diff
@@ -72,7 +72,7 @@ use the [Tokio] runtime, this means that:
 Once the application is instrumented, install the console CLI using
 
 ```shell
-cargo install tokio-console
+cargo install --locked tokio-console
```

Comment:
we should _probably_ make the use of `$` in shell examples consistent between the two READMEs --- I notice that the repo root readme always uses a `$` to indicate shell commands, while the `tokio-console` crate README doesn't. 

i have a _slight_ preference for omitting the `$` to make it easier to copy and paste the command into your shell, but it's not a strong preference.

in any case, we can address this in a follow-up change.